packages/extension-browser/src/devtools/views/controls/feedback-link.tsx

```diff
@@ -1,18 +1,86 @@
 import * as React from 'react';
+import { useState, useEffect } from 'react';
+
+import { Category } from 'hint/dist/src/lib/enums/category';
 
 import { getMessage } from '../../utils/i18n';
+import { Config, ErrorData } from '../../../shared/types';
+import metas from '../../../shared/metas.import';
 
 import ExternalLink from './external-link';
+import { evaluate } from '../../utils/inject';
+import escapeRegExp = require('lodash/escapeRegExp');
+
+const categories = [...new Set(metas.map((meta) => {
+    return (meta.docs && meta.docs.category || Category.other);
+}))];
+const { version } = require('../../../manifest.json');
+const template = require('../../../../../../../.github/ISSUE_TEMPLATE/2-bug-report-browser.md').default.replace(/---([\s\S]*)---/gm, '').trim();
+
+type Props = {
+    config: Config;
+    error?: ErrorData;
+    altText?: string;
+}
+
+const webhintVersionRegex = /__webhint\sversion:__/g;
+const browserVersionRegex = /__Browser\sversion:__/g;
+const urlRegex = /__URL\sfor\swhich\swebhint\sfailed:__/g;
 
-const openIssueUrl = 'https://github.com/webhintio/hint/issues/new?labels=type%3Abug&amp;template=2-bug-report-browser.md&amp;title=%5BBug%5D+Bug+description';
+const labels = 'type:bug';
+const title = '[Bug] Bug description';
+
+const getRegex = (value: string) => {
+    const baseRegexString = `\\*\\s\\[(x|\\s)]\\s({{value}})$`;
+
+    return new RegExp(baseRegexString.replace('{{value}}', value), 'gmi');
+};
 
 /**
  * Link to github to give feedback.
  */
-const FeedbackLink = () => {
+const FeedbackLink = ({ config, error, altText }: Props) => {
+    const [issueUrl, setIssueUrlUrl] = useState('');
+
+    const getLocation = () => {
+        const disabledCategories = config.disabledCategories || [];
+        const browserslists = config.browserslist === undefined ? '' : config.browserslist;
```

Comment:
Since we're just mapping to the empty string anyway...
```suggestion
        const browserslists = config.browserslist || '';
```